services/consensuscontext/test/harness.go

```diff
@@ -9,13 +9,27 @@ import (
 	"github.com/orbs-network/orbs-network-go/instrumentation/metric"
 	"github.com/orbs-network/orbs-network-go/services/consensuscontext"
 	"github.com/orbs-network/orbs-network-go/test/builders"
+	"github.com/orbs-network/orbs-spec/types/go/primitives"
 	"github.com/orbs-network/orbs-spec/types/go/protocol"
 	"github.com/orbs-network/orbs-spec/types/go/services"
 	"github.com/stretchr/testify/require"
 	"os"
 	"testing"
 )
 
+var federationNodePublicKeysForTest = []primitives.Ed25519PublicKey{
```

Comment:
we have a `KeyPairForTest` function somewhere - are you sure you need this?

Reply:
Comments for myself, removed.

Reply:
The test needs an array of public keys, not a single keypair, so added a func to `keys.ed25519.go`.

Reply:
Refactored all this part out of existence.